$NetBSD: patch-en,v 1.2 2011/07/11 09:57:18 ryoon Exp $

Use ligbcc builtin __clear_cache() declared in exec-all.h
instead of original __builtin___clear_cache() because
the latter one is not available in (at least) gcc 4.1.3.

--- tcg/arm/tcg-target.h.orig	2011-05-06 19:01:44.000000000 +0000
+++ tcg/arm/tcg-target.h
@@ -83,7 +83,7 @@ enum {
 static inline void flush_icache_range(unsigned long start, unsigned long stop)
 {
 #if QEMU_GNUC_PREREQ(4, 1)
-    __builtin___clear_cache((char *) start, (char *) stop);
+    __clear_cache((char *) start, (char *) stop); /* declared in exec-all.h */
 #else
     register unsigned long _beg __asm ("a1") = start;
     register unsigned long _end __asm ("a2") = stop;
